lib/cfd_trading/cfd_order_detail.dart

```diff
@@ -114,18 +110,16 @@ class _CfdOrderDetailState extends State<CfdOrderDetail> {
                               visible: confirm,
                               child: ElevatedButton(
                                   onPressed: () async {
-                                    // mock cfd has been closed.
-                                    Timer(const Duration(seconds: 2), () {
-                                      order.status = OrderStatus.closed;
-                                      cfdTradingService.persist(order);
-                                    });
+                                    // TODO: I think we don't need to update the UI state here, because it will be updated through polling the Cfd data from the backend
```

Comment:
Can we still do it, or at least trigger it from here? Otherwise the UX is not ideal because the user does an action and it takes a while before he sees it being changed in the frontend.

Reply:
See https://github.com/itchysats/10101/pull/231/commits/9d86a591083fc3bc1e349d0a7442fb4d09ac8277